src/main/java/org/skypro/skyshop/model/basket/ProductBasket.java

```diff
@@ -17,9 +17,8 @@ public ProductBasket(Map<UUID, Integer> products) {
     }
 
     public void addProduct(UUID productId) {
-        if (products.containsKey(productId)) {
-            products.put(productId, products.get(productId) + 1);
-        } else products.put(productId, 1);
+        products.computeIfPresent(productId, (k, v) -> (v + 1));
+        products.computeIfAbsent(productId, (k) -> (1));
```

Comment:
```suggestion
products.put(productId, products.computeIfAbsent(productId, k -> 1) + 1);
```
Вот так я имел ввиду. Т.е. если нет, то вернётся 1, иначе вернётся что есть. И далее это значение увеличиваем на 1.

Reply:
Дмитрий, затестила твой вариант. После первого добавления в корзине сразу два товара. Я что-то делаю не так?

Reply:
Извиняюсь, написал неправильно. Попробуй вот так
```suggestion
        products.put(productId, products.computeIfAbsent(productId, k -> 0) + 1);
```

Reply:
Теперь работает корректно, спасибо!